syntax = "proto3";

import "shared.proto";
import "entity.proto";

package quran;

service Image {
    rpc CreateImage (ImageEntity) returns (ImageEntity) {}
    rpc FindImageByAyahId (IDRequest) returns (ImageEntity) {}
}